set.mm

```diff
@@ -629242,6 +629242,963 @@ standardize vectors to a length (norm) of one, but such definitions require
+  different statements. If xxxv is preferred, then the versions in main should
+  be renamed as xxxvv since they all have more dv conditions). These weaker
+  versions can replace the proof steps occupied by their original counterparts
+  with the command "MINIMIZE_WITH xxxw /MAY_GROW". Since these theorems
+  follow the already present proof chain, automatic minimizations should not
+  cause significant proof changes or lengthenings, hence OLD proofs are not
+  necessary (and they would also be unpractical).
+
+  This list is not yet complete; the full one can be found at
+  https://github.com/GinoGiotto/set.mm/tree/ax-13-complete, where these
+  theorems are distinguishable from the others by searching for the
+  "(Contributed by Gino Giotto, 30-Dec-2023.)" tag.
+
+  To achieve the expected results, the following substitutions should be
+  additionally performed whenever possible:
+
+  - ~ sb8eu -> ~ sb8euv
+  - ~ sbie -> ~ sbiev
+  - ~ chvar -> ~ chvarfv
+  - ~ nfsb -> ~ nfsbv
+  - ~ nfmo -> ~ nfmov
+  - ~ sb8 -> ~ sb8v
+  - ~ chvarv -> ~ chvarvv
+  - ~ rgen2a -> ~ rgen2
+  - ~ spv -> ~ spvv
+
+  If everything goes well, at the end of the process, we should observe a
+  reduction of ~ ax-13 usage from more than 30,000 theorems, as shown in
+  https://github.com/GinoGiotto/set.mm/tree/ax-13-complete.
+
+  Additional information can be found in the mailing list at
+  https://groups.google.com/g/metamath/c/OB2_9sYgDfA.
+
+$)
+
+  ${
+    $d x y $.
+    $( Version of ~ nfae with a disjoint variable condition, which does not
+       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    nfaew $p |- F/ z A. x x = y $=
+      ( weq wal hbaev nf5i ) ABDAECABCFG $.
+
+    $( Version of ~ nfnae with a disjoint variable condition, which does not
+       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    nfnaew $p |- F/ z -. A. x x = y $=
+      ( weq wal nfaew nfn ) ABDAECABCFG $.
+  $}
+
+  ${
+    $d x y t $.  $d y ph $.
+    $( Version of ~ sb4b with a disjoint variable condition, which does not
+       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    sb4bw $p |- ( -. A. x x = t -> ( [ t / x ] ph
+                                              <-> A. x ( x = t -> ph ) ) ) $=
+      ( vy weq wal wn wi wsb wb nfvd wa nfnf1 nfan1 equequ2 imbi1d adantl albid
+      wnf pm5.74da syl albidv df-sb wex ax6ev a1bi 19.23v bitr4i 3bitr4g ) BCEZ
+      BFGZDCEZBDEZAHZBFZHZDFULUJAHZBFZHZDFZABCIURUKUPUSDUKULBSZUPUSJUKULBKVAULU
+      OURVAULLUNUQBVAULBULBMVAULBKNULUNUQJVAULUMUJADCBOPQRTUAUBABDCUCURULDUDZUR
+      HUTVBURDCUEUFULURDUGUHUI $.
+  $}
+
+  ${
+    $d x y $.
+    $( Version of ~ sb2 with a disjoint variable condition, which does not
+       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    sb2w $p |- ( A. x ( x = y -> ph ) -> [ y / x ] ph ) $=
+      ( weq wal wi wsb alim stdpc4 syl6com wn sb4bw biimprd pm2.61i ) BCDZBEZOA
+      FBEZABCGZFQPABEROABHABCIJPKRQABCLMN $.
+  $}
```

Comment:
This is (one direction of) https://us.metamath.org/mpeuni/sb6.html . So it can be proved from it and ~biimpri, but more importantly, I think it does not deserve its own statement.

Reply:
It's very strange that the minimizer doesn't catch this shortening, I even used [the stronger version](https://github.com/metamath/set.mm/pull/3619#issuecomment-1801361982) of it and still doesn't see it. However I should be careful before deleting one of these theorems, because I need to make sure that such deletion won't cause proof lengthenings (which would make the overall minimization process more difficult). For those 5 redundant ones with your mathbox it's easy because they are identical, but otherwise I need a bit of time to verify that.

Still that's a good catch, any other redundancy that you can find is appreciated.

Reply:
Ok, I checked and can safely get rid of `sb4bw` and `sb2w` at the minor cost of making `hbsb2w` a bit longer. I can also remove `hbsb2w` as well by lengthening `nfsb2w.` Again this just depends on what's the tolerance for proof lengthenings here.